src/utils/indexForSearch.ts

```diff
@@ -10,18 +10,36 @@ import { search } from "../lib/search"
  * This indexes an ElasticSearch cluster with collections data.
  */
 
+interface DebugRecord {
+  slug: string
+  reason: string
+  data: string
+}
+
 export const indexForSearch = async () => {
   let connection: Connection | undefined
-
+  const bogus_collections: DebugRecord[] = []
   try {
     const connectionArgs = databaseConfig()
     connection = await createConnection(connectionArgs)
 
     if (connection.isConnected) {
       const repository = getMongoRepository(Collection)
-
       const collections = await repository.find()
       for (const collection of collections) {
+        console.log(`Now processing ${collection.slug}`)
```

Comment:
Let's log out the slug of the collection we're looking at right away, so if we get a crash we can go find that record.

Reply:
Is this going to be really noisy & cause us to quickly reach a cap on any logging services?